hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteContainerCommandHandler.java

```diff
@@ -82,6 +82,11 @@ public void handle(final SCMCommand command,
     });
   }
 
+  @Override
+  public int getQueuedCount() {
```

Comment:
Since we are using directly this queue from threadpool, there is a chance one more task in progress running task right? For now it may be ok, I am just wondering we need to maintain our own counter to track all tasks including currently running task.

Reply:
You could be right. For replicate container command, it uses a slightly different implementation, so it does return the count of running + queued. However for others that use executors, the true queued work will be the queue_size + currently executing. However it will always be changing and will be out-of-date almost as soon as it arrives at SCM. I guess the queue size is a good approximation, but perhaps we should open another Jira to investigate making this more accurate in the future. What do you think?

Reply:
I am ok with separate JIRA. Just approved the current version.